Makefile

```diff
@@ -63,10 +67,6 @@ test: manifests generate fmt vet envtest ## Run tests.
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager main.go
 
-.PHONY: run
-run: manifests generate fmt vet ## Run a controller from your host.
-	go run ./main.go
```

Comment:
I'm personally a fan of this target, but I won't hold up these changes because of this.

Reply:
Mainly overwrote it to foster [functionality similar](https://github.com/operator-framework/rukpak/blob/77c1be1d734f82a0c07ca4b7893ac83f1033aa5b/Makefile#L107) to `RukPak`. Maybe we can keep this target as is and change `make run` => `make start`?

Reply:
I don't have a strong opinion but I thought the `run` command was automatically scaffolded out for us when kubebuilder initialized this project?